src/reports/components/fast-pass-report-summary.tsx

```diff
@@ -1,16 +1,31 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import * as React from 'react';
-import { FastPassOutcomeSummaryBar } from 'reports/components/fast-pass-outcome-summary-bar';
+import { FastPassReportResultData } from 'reports/components/fast-pass-report';
+import { OutcomeSummaryBar } from 'reports/components/outcome-summary-bar';
+import { OutcomeStats } from 'reports/components/outcome-type';
+import { allRequirementOutcomeTypes } from 'reports/components/requirement-outcome-type';
 
-export interface FastPassReportSummaryProps {}
+export interface FastPassReportSummaryProps {
+    results: FastPassReportResultData;
+}
 
 export class FastPassReportSummary extends React.Component<FastPassReportSummaryProps> {
     public render(): JSX.Element {
+        const stats: Partial<OutcomeStats> = {
+            //hardcoded sample stats, real numbers in future PR
+            fail: this.props.results.automatedChecks.cards.fail.length,
+            incomplete: this.props.results.automatedChecks.cards.unknown.length,
+            pass: this.props.results.automatedChecks.cards.pass.length,
+        };
+
         return (
             <>
                 <h2>Summary</h2>
-                <FastPassOutcomeSummaryBar />
+                <OutcomeSummaryBar
+                    outcomeStats={stats}
+                    allOutcomeTypes={allRequirementOutcomeTypes}
```

Comment:
I'd recommend having this component repeat the list explicitly, rather than relying on `allRequirementOutcomeTypes` - it's important for this component to control the order in which it specifies the outcome types, but if a future maintainer ever makes an update to `allRequirementOutcomeTypes` in `requirement-outcome-type.ts`, it might not be obvious to them that the order of the elements there will have a UI impact.

Reply:
great point, thank you!